src/parsing/nations_parsing.cpp

```diff
@@ -947,6 +947,7 @@ void scan_province_event(token_generator& gen, error_handler& err, scenario_buil
 		auto new_id = context.state.world.create_free_provincial_event();
 		auto fid = fatten(context.state.world, new_id);
 		fid.set_description(event_result.desc_);
+		fid.set_immediate_effect(event_result.immediate_);
```

Comment:
you need to make sure that the effects are created to take a province-province-approriate-from-slot. The existing logic wasn't expecting them to be added.

Reply:
see lines above